Migration_Scripts/calculate_pierce_points.py

```diff
@@ -34,6 +34,8 @@
     print('-----------------------------------------------------------------------------------------------------------------------')
     print('\n')
     sys.exit()
+    
+sys.path.insert(0, 'Path/to/smurfpy/Tools/TauP-2.5.0/bin')
```

Comment:
This can be removed. I will update to use obspy.taup.tau.TauPyModel.get_pierce_points_geo or equivalent.